README.md

```diff
@@ -37,16 +37,6 @@ be referred to by their respective pull-request numbers.
 What's useful to include?
 -------------------------
 
-The Valkey RFC format is not a strict format, but should include the following
-sections unless they are unnecessary for the proposal you are submitting.
-
-* Status and RFC number (the pull-request number).
-* Abstract. A few sentences describing the feature.
-* Motivation. What the feature solves and why the existing functionality is not
-  enough.
-* Design considerations. A description of the design constraints and
-  requirements for the proposal. Comparisons with similar features in other
-  projects.
-* Specification. A more detailed description of the feature, including why
-  certain details in the design have been chosen.
-* Links to related material such as issues, pull requests, papers, or other references.
+The Valkey RFC format is not a strict format, but should include details about
+various aspect of the engine, unless they are unnecessary for the proposal you 
+are submitting. Here's the [reference RFC template](REFERENCE.md) to get started.
```

Comment:
If the file is a template, I think it should be named TEMPLATE.md rather than REFERENCE.md.

I'm skeptical to deleting this overview description from the README. It provides a short overview that fits within the eyes of the reader without scrolling. Besides, I want to avoid a too formal process with a too strict mandatory format.

A common problem with templates is that people tend to leave a lot of template junk instead of deleting them, like the optional sections.

For these reasons, I think the template should be just an example/suggestion and not a mandatory thing to copy and edit.

Reply:
I started off with EXAMPLE.md but thought REFERENCE might be apt. I'm fine with `TEMPLATE.md` as well. Agaub this is just setting up devs to think through all the scenarios and don't need to conform to this.